### ISSUE TITLE
Some information about the current state of GTRCars - please delete if uninterested.

### ISSUE THREAD

**Issue by wren4777** [berkay2578/nfsw-server#2]
Dear Berkay

I'd like to start by apologising on behalf of everyone at DrivingPassion (formerly GTRCars) for the for your experience at our site. We completely ignored our fundamental value of freedom of speech when we bashed you on GTR.

With that said, we need you back at DrivingPassion. It is in a complete mess right now and we need some dedicated people to get the site back on track. If DP fails completely, me, ArturoPlayerOne and some other members are looking to create a new website for racing game modding and we invite you to take part in the creation of it in the event of DP's failure.

If you have any questions, please don't hesitate to contact me on Skype at bygh770.

Regards,

Haru (linuxfanatic)

**Comment by berkay2578** [comment]
I hanged around in GTRCars because there were some people who actually legit knew stuff and wasn't only all-talk (say hi leorblx!). I don't see what happened in my thread as an attack, it only reminded me why I'm almost always silent on the forums as most people are usually ignorant and just tries to yell at other people. That's the rule of the internet, not a problem with GTRCars members, not at all.

That being said, I don't see myself as a prophet that's magically able to restore anything. I didn't even know GTRCars moved to DrivingPassion. I only play with/modify stuff that I'm interested in and that I know for certain will add something to me. NFS:MW or NFS:C don't really interest me, and the only reason I do stuff for NFS:W is primarily because I played it during the early beta and also, it directly improves my skills. I also don't understand how I will be able to assist you guys though honestly, me hanging around the forums probably won't have much of an effect. Although, do hit me up on berkay2578[at]gmail[dot]com if you think I misunderstood you or something. 

Closing this as it's not relevant to the project.
